.github/workflows/build_cmake.yml

```diff
@@ -13,8 +13,6 @@ on:
 env:
   PLUGIN_NAME: QodeAssist
   QT_VERSION: 6.8.1
```

Comment:
Could you change to 6.8.2?

Reply:
Done in https://github.com/Palm1r/QodeAssist/pull/134.